static/js/controllers/contacts-content.js

```diff
@@ -147,24 +147,17 @@ var _ = require('underscore'),
         return fields;
       };
 
-      var shiftAndMarkContactPerson = function(children, parent) {
-        if (!children || !children.length) {
-          return;
-        }
-        var contactPersonId = parent.contact && parent.contact._id;
-        if (!contactPersonId) {
-          return;
+      var getPrimaryContact = function(doc) {
+        if (!doc || !doc.contact || !doc.contact._id) {
+          return $q.resolve();
         }
-        var primaryContactIdx = _.findIndex(children, function(child) {
-          return child.doc._id === contactPersonId;
-        });
-        if (primaryContactIdx < 0) {
-          return;
-        }
-        var primaryContact = children[primaryContactIdx];
-        primaryContact.isPrimaryContact = true;
-        children.splice(primaryContactIdx, 1);
-        children.unshift(primaryContact);
+        // Fetch the contact person from db instead of using the contents of the contact field.
+        // Because in 2017 we stop copying stuff in docs.
```

Comment:
Nothing yet! The storing within the doc of a copy of the contact doc. 
I'm refetching the contact, in preparation of that change.